worker-test/pom.xml

```diff
@@ -38,14 +38,32 @@
             <groupId>com.github.cafapi</groupId>
             <artifactId>caf-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
         <dependency>
             <groupId>com.github.workerframework</groupId>
             <artifactId>worker-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.rabbitmq</groupId>
+            <artifactId>amqp-client</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```

Comment:
The other tests use testng, let's not mix the two frameworks.
<img width="760" alt="image" src="https://github.com/WorkerFramework/worker-framework/assets/18459684/d0e6a739-6b7e-4435-a108-c14e576a57d6">